susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,93 @@
+{%- if grains['saltversion'] == '3000' and salt['pillar.get']('contact_method') not in ['ssh-push', 'ssh-push-tunnel'] and 
+       (not salt['pillar.get']('custom_info:mgr_avoid_venv_salt_minion') == 'True') %}
```

Comment:
Sorry to jump here so late in the game, I missed the rfc process.

There is however a problem with saltboot retail terminals. Saltboot, particularly dracut-saltboot, was released with support for salt bundle around October last year.

Initrd with saltboot is tied with deployed images on the terminal and we know our users are sometimes too cautious with deploying new image versions, so it is very well possible that some of our users are still using images from before saltboot dracut module understood salt bundle.

When this terminal is migrated to bundle, on next reboot dracut saltboot module will now recognize existing configuration and will try to register terminal as a new one.

Unfortunately it is not easy to determine is used initrd understands bundle or not, so I think for time being we should not migrate terminals to the bundle automatically (they are migrated automatically with new images).

To detect if minion is a terminal, look for `/etc/ImageVersion` file. If it is present, it is a retail terminal and we should IMO skip automigrating it.

Reply:
@nadvornik any thoughts?

Reply:
I included the check for `/etc/ImageVersion`.

Reply:
I think it could work with old images by accident - this PR deletes only `susemanager.conf` and the old retail images
put `master` entry in different file, so the old configuration is preserved and dracut saltboot continues to work.

But I agree that checking `/etc/ImageVersion` is better.

Reply:
I don't quite get the part about having `master:` in a different file @nadvornik, what change is still needed? 
As far as I understood, we want to abort saltbooted clients. I therefore think it's enough to remove `/etc/salt/minion.d/susemanager.conf`.